.pre-commit-config.yaml

```diff
@@ -7,7 +7,7 @@ repos:
           - --fix
           - --exit-non-zero-on-fix
           - --line-length=120
-          - --ignore=E731,E501,W605,T201
+          - --ignore=E731,E501,W605,T201,UP007
```

Comment:
Needed as ruff wants `Optional[str]` to be written `str | None`, which is not supported by `typer`, were we use it for example 
```python
include: Annotated[
        Optional[list[str]],	        Optional[list[str]],
        typer.Option(	        typer.Option(
            "--include",	            "--include",
            "-i",	            "-i",
            help=f"Specify which resources to deploy, available options: {_AVAILABLE_DATA_TYPES}",	            help=f"Specify which resources to deploy, available options: {_AVAILABLE_DATA_TYPES}",
        ),	        ),
    ] = None,	    ] = None,
```